qiskit_experiments/framework/base_experiment.py

```diff
@@ -265,33 +265,91 @@ def _finalize(self):
         """
         pass
 
-    def _run_jobs(self, circuits: List[QuantumCircuit], **run_options) -> List[Job]:
-        """Run circuits on backend as 1 or more jobs."""
+    def _max_circuits(self, backend: Backend = None):
+        """
+        Calculate the maximum number of circuits per job for the experiment.
+        """
+
+        # set backend
+        if backend is None:
+            if self.backend is None:
+                raise QiskitError("A backend must be provided.")
+            backend = self.backend
+        self.backend = backend
         # Get max circuits for job splitting
-        max_circuits_option = getattr(self.experiment_options, "max_circuits", None)
+        max_circuits_option = getattr(self.experiment_options,
+                                      "max_circuits",
+                                      None)
         max_circuits_backend = self._backend_data.max_circuits
+
         if max_circuits_option and max_circuits_backend:
-            max_circuits = min(max_circuits_option, max_circuits_backend)
+            return min(max_circuits_option, max_circuits_backend)
         elif max_circuits_option:
-            max_circuits = max_circuits_option
+            return max_circuits_option
         else:
-            max_circuits = max_circuits_backend
+            return max_circuits_backend
+
+    def job_info(self, backend: Backend = None):
+        """
+        Get information about job distribution for the experiment on a specific
+        backend.
+
+        Args:
+            backend (Backend): The backend for which to get job distribution
+            information.
+
+        Returns:
+            dict: A dictionary containing information about job distribution.
+                - "Total number of circuits in the experiment": Total number of
+                circuits in the experiment.
+                - "Maximum number of circuits": Maximum number of circuits in
+                one Job
+                - "Total number of jobs": Number of jobs needed for
+                distribution.
```

Comment:
This is still not building correctly, I suggest this:
```suggestion
            dict: A dictionary containing information about job distribution.

                - "Total number of circuits in the experiment": Total number of
                  circuits in the experiment.

                - "Maximum number of circuits per job": Maximum number of circuits in
                  one job based on backend and experiment settings.

                - "Total number of jobs": Number of jobs needed to run this experiment
                  on the currently set backend.
```